packages/typescript/package.json

```diff
@@ -53,7 +53,7 @@
     "vitest": "^2.1.1"
   },
   "peerDependencies": {
-    "@metamask/eslint-config": "^13.0.0",
+    "@metamask/eslint-config": "workspace:^",
```

Comment:
Does this show up on npm as `workspace:^`, or as a specific version string? It looks like it's the latter for snaps packages? Does that work with no additional configuration?

Reply:
Yarn replaces `workspace:^` with the actual version before publishing, so this works without any additional configuration.

Reply:
Nice! Good to know.